### DISCUSSION TITLE
Open issue on support repo for a new branding repo

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#599] [First-Timers] [answered]
### What to do?
🟩 
It would be great, if someone could open an issue on the [support](https://github.com/EddieJaoudeCommunity/support/) repository for 'making' a new github repository for the community called `branding`, where our community's artwork, emojis etc. can reside, open for contributions, and suggestions.

### More on the ISSUE:

Our discord server lacks some custom emotes :P
Making a few more might add value to the server. Moreover, this might also boost inclusivity in the community, opening the community to contributions from not only programmers and enthusiasts, but also artists, designers, hobbyists. 

Moreover, adding this new repository would help us create customisable community art, which could be used on the website and on the discord server. We could also use this to discuss on community slogans etc. 🤔 

### Additional context:

(this all started with me badly editing the `eddie` emoji xD)

![Screenshot 2021-01-01 at 8 56 33 PM](https://user-images.githubusercontent.com/62864373/103441414-da28f780-4c73-11eb-8192-6258d534c4ad.png)

**Comment by eddiejaoude** [comment]
I think this is a great idea 👍 - all the source files can be kept there if people want to make some changes - probably need some rules around this, but lets hope people play nice 🤓

**Reply by Vyvy-vi** [reply]
I think we could add some additional pointers in the default COC for that repo 🤔 
Or perhaps make another file for that(or add in README)

Though hopefully, things will go neat and clean :)

**Comment by Vyvy-vi** [comment, marked as answer]
An issue for this has been opened by @SuNiL-Chau
at [Support#615](https://github.com/EddieJaoudeCommunity/support/issues/615)

**Reply by eddiejaoude** [reply]
Great! thank you all 👍
